ideal_response/ideal_response_functions.R

```diff
@@ -72,6 +72,7 @@ norm_p_over_frequency <- function(f, f_ulco, f_upper) {
 }
 
 down_sample_1s <- function(ideal_response_1_s, duration, offset) {
+  if (duration != 1) {   # down-sampling is ignored for 1s data
   ideal_response_1_s <- thicken(
```

Comment:
As an if statement is added here, we want to indent the following lines by two spaces.

Reply:
Spacing after "if" statement is changed. Thank you.

Reply:
The arguments for "thicken" should be indented by an additional two spaces for readability

Reply:
Indented the arguments "thicken" by additional two spaces. Thank you.